src/graphql/subgraph/vault/userStatsQuery.graphql

```diff
@@ -0,0 +1,29 @@
+query UserStats($user: Bytes! $vaultAddress: String! $first: Int!) {
+  allocatorStats_collection(
+    interval: day
+    where: { allocator_: { address: $user, vault: $vaultAddress } }
+    first: $first
+  ) {
+    timestamp
+    earnedAssets
+    totalAssets
+  }
+  exitRequestStats_collection(
+    interval: day
+    where: { exitRequest_: { receiver: $user, vault: $vaultAddress } }
+    first: $first
```

Comment:
If there is `first`, should we add $skip as well?

Reply:
https://github.com/stakewise/v3-sdk/pull/176/commits/10c41098aed5c24647e5a3dbe409d53efd023dce#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L808